frontend/src/scenes/insights/LineGraph.js

```diff
@@ -173,144 +173,161 @@ export function LineGraph({
-            options:
-                type !== 'doughnut'
-                    ? {
-                          responsive: true,
-                          maintainAspectRatio: false,
-                          scaleShowHorizontalLines: false,
-                          tooltips: {
-                              enabled: true,
-                              intersect: false,
-                              mode: 'nearest',
-                              // If bar, we want to only show the tooltip for what we're hovering over
-                              // to avoid confusion
-                              ...(type !== 'bar' ? { axis: 'x' } : {}),
-                              bodySpacing: 5,
-                              position: 'nearest',
-                              yPadding: 10,
-                              xPadding: 10,
-                              caretPadding: 0,
-                              displayColors: false,
-                              backgroundColor: '#1dc9b7',
-                              titleFontColor: '#ffffff',
-                              labelFontSize: 23,
-                              cornerRadius: 4,
-                              fontSize: 12,
-                              footerSpacing: 0,
-                              titleSpacing: 0,
-                              footerFontStyle: 'italic',
-                              callbacks: {
-                                  label: function (tooltipItem, data) {
-                                      let entityData = data.datasets[tooltipItem.datasetIndex]
-                                      if (entityData.dotted && !(tooltipItem.index === entityData.data.length - 1)) {
-                                          return null
-                                      }
-                                      const label = entityData.chartLabel || entityData.label || ''
-                                      const formattedLabel = entityData.action
-                                          ? formatLabel(label, entityData.action)
-                                          : label
-                                      return (
-                                          (formattedLabel ? formattedLabel + ' — ' : '') +
-                                          tooltipItem.yLabel.toLocaleString() +
-                                          (percentage ? '%' : '')
-                                      )
+            hover: {
+                mode: 'nearest',
+                onHover(evt) {
+                    if (onClick) {
+                        const point = this.getElementAtEvent(evt)
+                        if (point.length) {
+                            evt.target.style.cursor = 'pointer'
+                        } else {
+                            evt.target.style.cursor = 'default'
+                        }
+                    }
+                },
+            },
+            onClick: (_, [point]) => {
+                if (point && onClick) {
```

Comment:
Good question. I think it's a follow-up enhancement because the people modal is at the moment very date-based.

We might need to change the date filtering in sessions first though as well.

Reply:
Investigated, opened https://github.com/PostHog/posthog/issues/3472 - penny for your thoughts on this.